CODEOWNERS

```diff
@@ -40,12 +43,14 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # IE Tests:
 /inference-engine/tests/  @openvinotoolkit/openvino-ie-tests-maintainers
 /inference-engine/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers
+/inference-engine/tests/functional/inference_engine/ngraph_reader  @GlebKazantaev @ichuraev
```

Comment:
probably should be under the nGraph group below

Reply:
agree